.github/workflows/tests.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
             ~/.cargo/git
       - name: Install
         run: |
-          make install
+          make install-cpu
```

Comment:
Simple server tests are running on non accelerated hardwared, therefore cannot install the kernels.